<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="Statistique">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Personnel">
       <attribute name="title">
        <string>Employees</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Liste du personnel :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Last Name :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditNom"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>First Name :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditPrenom"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pseudo :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditPseudo"/>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Reset password</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Phone :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditTelFixe"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Cell phone :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditTelPort"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Email :</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditMail"/>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Address :</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditAdresse"/>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Postal Code :</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditCP"/>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>City :</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditVille"/>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>User type :</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QRadioButton" name="radioButtonController">
          <property name="text">
           <string>Controller</string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QRadioButton" name="radioButtonManager">
          <property name="text">
           <string>Manager</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Iban :</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEditIban"/>
        </item>
        <item row="12" column="1" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="butAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="butUpdate">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="butDel">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" rowspan="12">
         <widget class="QListWidget" name="listWidgetPersonnels"/>
        </item>
       </layout>
       <zorder>listWidgetPersonnels</zorder>
       <zorder>label</zorder>
       <zorder>label_2</zorder>
       <zorder>lineEditNom</zorder>
       <zorder>lineEditPrenom</zorder>
       <zorder>label_3</zorder>
       <zorder>label_4</zorder>
       <zorder>lineEditPseudo</zorder>
       <zorder>label_5</zorder>
       <zorder>label_6</zorder>
       <zorder>label_7</zorder>
       <zorder>label_8</zorder>
       <zorder>label_9</zorder>
       <zorder>label_10</zorder>
       <zorder>lineEditTelFixe</zorder>
       <zorder>lineEditTelPort</zorder>
       <zorder>lineEditMail</zorder>
       <zorder>lineEditAdresse</zorder>
       <zorder>lineEditCP</zorder>
       <zorder>lineEditVille</zorder>
       <zorder>radioButtonManager</zorder>
       <zorder>radioButtonController</zorder>
       <zorder>label_11</zorder>
       <zorder>pushButton</zorder>
       <zorder>label_12</zorder>
       <zorder>lineEditIban</zorder>
      </widget>
      <widget class="QWidget" name="GestionProduit">
       <attribute name="title">
        <string>Gestion</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" rowspan="6">
         <widget class="QSplitter" name="splitter_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>Liste des produits :</string>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidgetProduit"/>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Rayon :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="cbRayon"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonAddRayon">
          <property name="text">
           <string>Ajouter rayon</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonDelRayon">
          <property name="text">
           <string>Supprimer un rayon</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Categorie :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBoxCat"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButtonAddCat">
          <property name="text">
           <string>Ajouter categorie</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButtonDelCat">
          <property name="text">
           <string>Supprimer categorie</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Nom du produit :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEditNomProd"/>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>Ajouter produit</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButtonDel">
          <property name="text">
           <string>Supprimer produit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Visite">
       <attribute name="title">
        <string>Visite</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Controleur :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Personne a visiter/derniere visite :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidgetControleur"/>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QListWidget" name="listWidgetProducteur"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Prochaine visite :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Date :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" rowspan="5">
         <widget class="QListWidget" name="listWidgetVisite"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Heure debut :</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QTimeEdit" name="timeEditDebut"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Heure fin :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QTimeEdit" name="timeEditFin"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Commentaire :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="2" colspan="2">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item row="7" column="2" colspan="2">
         <widget class="QPushButton" name="addVisite">
          <property name="text">
           <string>Ajouter Visite</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Stat">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
